liberate/models/ictfile

```diff
@@ -0,0 +1,1040 @@
+process "GPDK45" {
```

Comment:
No idea what this is but I'm assuming it's an "input" file.

If it is an "output" file, maybe we should add it to the .gitignore. Outputs should be ignored to keep diffs more readable